subsys/net/lib/websocket/Kconfig

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig WS
```

Comment:
I'd say this is too cryptic. Despite the long sub-option names, I'd suggest to call this at least WEBSOCK

Reply:
I think WEBSOCK is too long as we would need to change the API names too and then this WEBSOCK is way too long. If we really want to have more characters here, then perhaps WSS (websocket service) or similar could be used.

Reply:
My concern is that prj.conf remained readable. To that end, I'd say it might be possible to untie API prefixes from Kconfig values, and actually to have CONFIG_WEBSOCK, but still CONFIG_WS_HTTP_SERVER_NUM_URLS. Of course, that's just an opinion.

Reply:
I changed the main config from CONFIG_WS to CONFIG_WEBSOCKET, this is indeed more readable now. I left the other variables as is as you suggested.

Reply:
WS is indeed too cryptic, is this WEBSERVICE, WEBSITE, WORKSHOP, WALLSTREET or maybe WORKSTATION?

Please make it clear by using WEBSOCKET, I do not think it is too long